src/client/bridge-data.ts

```diff
@@ -74,18 +85,29 @@ export interface BridgeDataFieldGetters {
   ): Promise<bigint[]>;
 
   /*
-  @dev This function should be implemented for async bridges. It should return the date at which the bridge is expected to be finalised.
-  @dev For limit orders this should be the expiration.
+  @dev This function should be implemented for async bridges
+  @param interactionNonce A globally unique identifier of a given DeFi interaction
+  @return The date at which the bridge is expected to be finalised (for limit orders this should be the expiration)
   */
   getExpiration?(interactionNonce: bigint): Promise<bigint>;
 
+  /*
+  @dev This function should be implemented for async bridges
+  @param interactionNonce A globally unique identifier of a given DeFi interaction
+  @return A boolean indicating whether the interaction has been finilised
+  */
   hasFinalised?(interactionNonce: bigint): Promise<boolean>;
 
   /*
-  @dev This function should be implemented for all bridges are stateful. It should return the expected value 1 year from now of outputAssetA and outputAssetB
-  @dev given inputValue of inputAssetA and inputAssetB
+  @dev This function should be implemented for all bridges that are stateful
+  @param inputAssetA A struct detailing the first input asset
+  @param inputAssetB A struct detailing the second input asset
+  @param outputAssetA A struct detailing the first output asset
+  @param outputAssetB A struct detailing the second output asset
+  @param auxData Arbitrary data to be passed into the bridge contract (slippage / nftID etc)
+  @param inputValue Amount of inputAssetA (and inputAssetB if used) provided on input
+  @return The expected value 1 year from now of outputAssetA and outputAssetB
```

Comment:
Specify the unit. %, bps something else?

Reply:
In this case it's absolute amounts of output assets. Will clarify.